src/screens/Search/Result.tsx

```diff
@@ -57,10 +70,11 @@ export type ResultDataProps = {
   translation: string,
 }
 
-export type ResultProps = PressableProps & ResultDataProps & {
+export type ResultProps = PressableProps &
+ResultDataProps & {
```

Comment:
[style] another case for ESLint, should have the & before it and indented, or b eon same line